packages/node/src/evm/templates/template-application.ts

```diff
@@ -22,12 +22,9 @@ function applyTemplate(
 ): ClientRequest<ApiCall> {
   return {
     ...request,
-    // NOTE: template attributes can be overwritten by the request attributes
-    designatedWallet: request.designatedWallet || template.designatedWallet,
+    // providerId and endpointId will either be request or a template attribute
+    providerId: request.providerId || template.providerId,
```

Comment:
I'm not sure why this wasn't applying the `providerId` before

Reply:
Weird. I took a quick look and can't find anything else that would have applied it for short/regular requests.